unused_deps/unused_deps.go

```diff
@@ -291,6 +291,8 @@ func printCommands(label string, deps map[string]bool) (anyCommandPrinted bool)
 				if hasRuntimeComment(str) {
 					fmt.Printf("buildozer 'move deps runtime_deps %s' %s\n", str.Value, label)
 				} else {
+					// add dep's exported dependencies to label before removing dep
+					fmt.Printf("buildozer \"add deps $(%s query 'labels(exports, %s)' | tr '\\n' ' ')\" %s\n", *buildTool, str.Value, label)
```

Comment:
I tried to locate a utility function for obtaining a label's attribute but was unable to find one, so I used a bazel query invocation here. It'd be better to avoid the query. I'd appreciate any pointers on this in case I missed something.